bundles/net.tourbook.cloud/src/net/tourbook/cloud/strava/StravaUploader.java

```diff
@@ -515,17 +519,21 @@ public IStatus run(final IProgressMonitor monitor) {
          @Override
          public void done(final IJobChangeEvent event) {
 
-            if (PlatformUI.isWorkbenchRunning() && event.getResult().isOK()) {
+            if (PlatformUI.isWorkbenchRunning()) {
+
+               final String infoText = event.getResult().isOK()
+                     ? NLS.bind(Messages.Dialog_UploadToursToStrava_Message,
+                           numberOfUploadedTours[0],
+                           numberOfTours - numberOfUploadedTours[0])
```

Comment:
Instead of `NLS.bind()` I'm using nowadays `"xyz %d".formatted(intValue)`

Reply:
I think it's a `.formatted()` is a good alternative but I just got "bitten" with it because in the below case, I think it's better to use `NLS.bind()` as part of the string is repeated (`{1}`)and `.formatted()` doesn't support multiple identical objects without repeating those objects, which makes the java code redundant.

`Log_UploadToursToStrava_003_ActivityLink = {0} -> Uploaded Activity Link: <br><a href="https://www.strava.com/activities/{1}">https://www.strava.com/activities/{1}</a></br>`

Reply:
For now, I will keep the existing `NLS.bind()` unless you tell me to switch them all to `.formatted()`

Reply:
> Good to know, so we shouldn't use `NLS.bind()` anymore ?

It's up to you, it is an analternative which supports better formatting options for number values

Reply:
> `.formatted()` doesn't support multiple identical objects without repeating those objects

Have you tested it with the `[argument_index$]` parameter, I've never used it

https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/Formatter.html#syntax

Reply:
I will test it, good idea 👍